lab-james/test/fs-read-test.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const fs = require('fs');
+const expect = require('chai').expect;
+const Buffer = require('buffer').buffer;
+const fsReadModule = require(`${__dirname}/../lib/fs-read.js`);
```

Comment:
So you dont actually have to put in a template literal in here, a string path is fine.